wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/Plugin.java

```diff
@@ -1,22 +1,19 @@
 package com.squareup.wire.protocwire;
 
+import com.google.protobuf.ByteString;
 import com.google.protobuf.CodedOutputStream;
 import com.google.protobuf.DescriptorProtos.FileDescriptorProto;
+import com.google.protobuf.Descriptors;
 import com.google.protobuf.Descriptors.DescriptorValidationException;
 import com.google.protobuf.Descriptors.FileDescriptor;
+import com.google.protobuf.ExtensionRegistry;
 import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest;
 import com.google.protobuf.compiler.PluginProtos.CodeGeneratorResponse;
-import com.google.protobuf.compiler.PluginProtos.CodeGeneratorResponse.File;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.io.StringWriter;
-import java.io.Writer;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
I don't recall doing this. Is there a default setting in IntelliJ to collapse imports to a wildcard if there are too many from the same package?

Reply:
This is done in the settings. 
<img width="979" alt="image" src="https://user-images.githubusercontent.com/7771562/186209531-fab79edc-f2e6-4335-8cf4-280a4abb4e0b.png">

You're gonna have to do it for the Kotlin and Java tabs :(

Reply:
I'm not a fan of the wildcard imports also 😂

Reply:
Me neither! I only noticed this when I was reviewing the diff upon opening this PR. I'll change it back and configure the IDE to not do such foolish things...

Reply:
Fixed